examples/linuxbrige_eth1_up.yml

```diff
@@ -22,3 +22,12 @@ interfaces:
           stp-hairpin-mode: false
           stp-path-cost: 100
           stp-priority: 32
+          vlan:
```

Comment:
Please create new example file for this.

Reply:
Done.